docs/csharp/language-reference/compiler-options/language.md

```diff
@@ -121,13 +121,14 @@ The following table lists the minimum versions of the SDK with the C# compiler t
 
 ## Nullable
 
-The **Nullable** option lets you specify the nullable context.  The default value for this option is `disable`.
+The **Nullable** option lets you specify the nullable context. It can be set in the project's configuration using the `<Nullable>` tag:
 
 ```xml
 <Nullable>enable</Nullable>
 ```
 
 The argument must be one of `enable`, `disable`, `warnings`, or `annotations`. The `enable` argument enables the nullable context. Specifying `disable` will disable the nullable context. When providing the `warnings` argument the nullable warning context is enabled. When specifying the `annotations` argument, the nullable annotation context is enabled.
+<br/>When there's no value set, the default value `disable` is applied, however **note** that the .NET 6 templates are by default provided with the **Nullable** value set to `enable`.
```

Comment:
I like the text. This would follow our styles and use our extensions a bit:

```suggestion

> [!NOTE]
> When there's no value set, the default value `disable` is applied, however the .NET 6 templates are by default provided with the **Nullable** value set to `enable`.
```